frontstage/i18n/translations.py

```diff
@@ -0,0 +1,49 @@
+import logging
+from json import load
+from structlog import wrap_logger
+from frontstage import app
+from os.path import join
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+class Translate:
+
+    def __init__(self, translation_file):
+        translation_default_path = join(app.root_path, 'i18n')
+        self.translation_path = app.config.get('TRANSLATIONS_PATH', translation_default_path)
+        default_locale = 'en_GB'
+        self.locale = app.config.get('DEFAULT_LOCALE', default_locale)
+
+        try:
+            if not translation_file.endswith('.json'):
+                translation_file += '.json'
+
+            file_name = join(self.translation_path, translation_file)
+            with open(file_name) as f:
+                self.translations = load(f)
+        except FileNotFoundError as exc:
+            logger.error(f'Instantiating translation with file "{file_name}"')
```

Comment:
We shouldn't be using f strings in logs it should be key value pairs

Reply:
Also, would you be able to change this error message please to make it clear that the file isn't there, just to make it easier for anyone seeing the error in logs.

Reply:
@rhiGriff Can you provide an example - naturally I have no access to the page you linked.  Any chance of a snippet to demonstrate what you might write?

I'm relatively sure that if all logging needs to be KV of some sort, that Frontstage basically needs all log lines rewritten, so might need a ticket for that too.

Reply:
I did some work on it before but yeah it will need looking at again.

Here is an example:

 logger.info('User account is locked on the OAuth2 server', party_id=party_id)

https://github.com/ONSdigital/ras-frontstage/blob/91d1ee655f037b6c817fcedd4f5f99279b5ca140/frontstage/views/sign_in/sign_in.py#L37